core/services/chainlink/config.go

```diff
@@ -42,6 +42,33 @@ type Config struct {
 	Solana solcfg.TOMLConfigs `toml:",omitempty"`
 
 	Starknet stkcfg.TOMLConfigs `toml:",omitempty"`
+
+	Aptos []*RawConfig `toml:",omitempty"`
+}
+
+// RawConfig is the config used for chains that are not embedded.
+type RawConfig map[string]any
+
+// RawConfigs is a list of RawConfig.
+type RawConfigs []RawConfig
+
+// ValidateConfig returns an error if the Config is not valid for use, as-is.
+func (c *RawConfig) ValidateConfig() error {
```

Comment:
You can test this by adding an invalid value here: https://github.com/smartcontractkit/chainlink/blob/a830fe093f4642236a89ac73bf9eefe5008898bc/core/services/chainlink/config_test.go#L1251-L1252

Notice the error messages are formatted. You should return one of these well known errors that include the field name: https://github.com/smartcontractkit/chainlink/blob/a830fe093f4642236a89ac73bf9eefe5008898bc/core/chains/evm/config/toml/config.go#L923-L933

Reply:
thanks for this suggestion, this is much more elegant way. noted on testing as well